pyproject.toml

```
@@ -18,6 +18,7 @@ classifiers = [
 ]
 requires-python = ">=3.10"
 dependencies = [
+    "runtime-introspect>=0.2.0",
```

I feel everyone involved (myself included, of course) would feel more confortable with a loosely pinned requirement here

```suggestion
    "runtime-introspect~=0.2.0",
```